src/variations.json

```diff
@@ -4,38 +4,18 @@
     "prefs": {
       "browser.contentblocking.features.standard": "-tp,tpPrivate,cookieBehavior0,-cm,-fp",
       "browser.contentblocking.features.strict": "tp,tpPrivate,cookieBehavior4,-cm,-fp",
-      "privacy.trackingprotection.fingerprinting.enabled": false,
-      "privacy.trackingprotection.cryptomining.enabled": false
-    }
-  },
-
-  "FP": {
-    "weight": 1,
-    "prefs": {
-      "browser.contentblocking.features.standard": "-tp,tpPrivate,cookieBehavior0,-cm,fp",
-      "browser.contentblocking.features.strict": "tp,tpPrivate,cookieBehavior4,-cm,fp",
-      "privacy.trackingprotection.fingerprinting.enabled": true,
-      "privacy.trackingprotection.cryptomining.enabled": false
+      "privacy.trackingprotection.cryptomining.enabled": false,
+      "privacy.trackingprotection.fingerprinting.enabled": false
     }
   },
 
   "CM": {
```

Comment:
Should we revisit this branch name since this is now a "control and treatment" type study?

Reply:
I'll ask whomever is on data once we figure out who that is